images.CI/lint-shebang.ps1

```diff
@@ -0,0 +1,38 @@
+param(
+    [Parameter(Mandatory)] [string] $path,
+    [Parameter(Mandatory)] [string] $pattern
+)
+
+$ErrorActionPreference = "Stop"
+
+function Validate-Scripts {
+    Param (
+        [Parameter(Mandatory=$true)]
+        [string[]]$Path,
+        [Parameter(Mandatory=$true)]
+        [string]$Pattern
```

Comment:
May be `$Pattern` -> `$ExpectedShebang`?

Reply:
Thank you, I have fixed this.